<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-198-35" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" tests="6" failures="0" timestamp="2023-04-15T06:57:07 UTC" time="60.283" errors="0">
  <testcase name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.043" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.045" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429" time="5.062" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504" time="2.050" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_502" time="13.041" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_503" time="40.042" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
